stock_picking_delivery_link/models/stock_picking.py

```diff
@@ -55,7 +55,7 @@ def _pre_put_in_pack_hook(self, move_line_ids):
         else:
             return res
 
-    def _set_delivery_package_type(self):
+    def _set_delivery_package_type(self, batch_pack=False):
         self.ensure_one()
         res = super()._set_delivery_package_type()
```

Comment:
but then you should probably pass it no?

Reply:
yub, my bad